src/shibuya.js

```diff
@@ -123,9 +123,9 @@ const main = async () => {
             },
         },
         {
-            V2: [
+            V2: [ 
```

Comment:
Add the DescendOrigin instructions here to see if the call could pass. If we don’t write it down, no reader would know that a DescendOrigin happens first, and we have to do `V2.length + 1` for the fees.

Reply:
DescendOrigin can’t be explicitly added here, so we are skipping the first instruction.